### PR TITLE
Misssing imports and object declaration

### PR DESCRIPTION
**Pull request by peterdays**

### CODE REVIEW

**Comment by peterdays** [comment]
Should the loss function `poisson_multinomial_torch` and also the weight decay one, be added to borzoi-pytorch, and imported from there, instead of having these files copied here? @johahi

**Comment by johahi** [comment]
Hi, sorry for the delay - I prefer this repo to be standalone for now

**Comment by peterdays** [comment]
I see, either way this PR adds the imports without external dependencies, so feel free to accept it or reject!
